### ISSUE TITLE
[feature] Unfortunate HTML output template placeholder

### ISSUE THREAD

**Issue by kvid** [wireviz/WireViz#377]
HTML output templates with placeholders for generated text is a new feature introduced in v0.4. All placeholders are named `<!-- %something% -->`, except one placeholder called `"sheetsize_default"` only.
1. Why is it called `sheetsize_default` and not `sheetsize` or `template_sheetsize`? The replacement text is fetched from `metadata.template.sheetsize`.
2. Why is it not called `<!-- %template_sheetsize% -->` to be consistent with the other placeholders?
3. I guess the quote characters was needed to include in the search term to avoid false hits, and therefore re-inserted around the fetched value to keep the correct HTML syntax. This seems like a hack for this placeholder only, and would not be needed if using my suggestion in point 2 above. Then `<!-- %template_sheetsize% -->` could be placed within the quote characters in the template, but the quote characters need not be replaced, and special treatment of this placeholder is no longer needed. The code would be cleaner, and a documentation description would be easier to understand because all placeholders are treated equally.

I suggest replacing https://github.com/wireviz/WireViz/blob/v0.4/src/wireviz/wv_html.py#L104-L106 with a simple entry
```py
        "<!-- %template_sheetsize% -->": metadata.get("template", {}).get("sheetsize", ""),
```
inserted together with the others at https://github.com/wireviz/WireViz/blob/v0.4/src/wireviz/wv_html.py#L88
and then update the placeholder to `<!-- %template_sheetsize% -->` in `templates/din-6771.html`

**Comment by martinrieder** [comment]
I propose to use the default placeholder pattern that is used in Jinja. It would allow using this as a templating engine that is more powerful than simple replacements:

- block_start_string
  The string marking the beginning of a block. Defaults to `'{%'`.
- block_end_string
  The string marking the end of a block. **Defaults to `'%}'`.**
- variable_start_string
  The string marking the beginning of a print statement. **Defaults to `'{{'`.**
- variable_end_string
  The string marking the end of a print statement. **Defaults to `'}}'`.**
- comment_start_string
  The string marking the beginning of a comment. Defaults to `'{#'`.
- comment_end_string
  The string marking the end of a comment. **Defaults to `'#}'`.**

[Jinja Template Designer Documentation](https://jinja.palletsprojects.com/en/3.0.x/templates/)

**Comment by kvid** [comment]
@formatc1702 - Is my original issue here trivial, or might there be bad side-effects?

The suggestion by @martinrieder is also valid, but that must be handled later.

**Comment by kvid** [comment]
Closed after releasing v0.4.1.
